spec/models/book_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'rails_helper'
+require 'faker'
+
+describe Book, type: :model do
+  it do
+    is_expected.to validate_presence_of(:genre)
+  end
+  it do
+    is_expected.to validate_presence_of(:author)
+  end
+  it do
+    is_expected.to validate_presence_of(:title)
+  end
+  it do
+    is_expected.to validate_presence_of(:publisher)
+  end
+  it do
+    is_expected.to validate_presence_of(:year)
+  end
+
+  subject(:Book) do
```

Comment:
remove this subject also, when you do declare variables with let or subject, use snake case with all the letters in lowercase